pkgs/by-name/fu/furnace/package.nix

```diff
@@ -64,6 +65,7 @@ stdenv.mkDerivation (finalAttrs: {
       rtmidi
       SDL2
       libX11
+      libGL
```

Comment:
Please put this into a `lib.optionals withGL`.

And move the `libX11` above it into the `lib.optionals stdenv.hostPlatform.isLinux` list. Seems like that was commited via staging without taking a closer look at the derivation :s.